demo/index.js

```diff
@@ -163,7 +168,11 @@ import { OpenSheetMusicDisplay } from '../src/OpenSheetMusicDisplay/OpenSheetMus
             openSheetMusicDisplay.cursor.hide();
         });
         showCursorBtn.addEventListener("click", function() {
-            openSheetMusicDisplay.cursor.show();
+            if (openSheetMusicDisplay.cursor) {
+                openSheetMusicDisplay.cursor.show();
+            } else {
+                console.info("can't show cursor, cursor was disabled by drawingParameters!");
```

Comment:
Ok, this is suuuper picky now, but maybe we want the message to start with a capital letter if it is ended as a sentence 🙈

Reply:
As a fellow word nerd, I agree ;)